en/vespa8-release-notes.html

```diff
@@ -596,10 +580,18 @@ <h3 id="request-headers-for-remote-in-access-log">Request headers controlling re
 </ul>
 
 
-<!--
 <h2 id="other-changes">Other changes</h2>
+<!--
 <p>
-Check:
+TODO Check:
 * VESPA-7875 Move XML files to top-level inside application package
 </p>
 -->
+<h3 id="qrserver-service-name">The "qrserver" service name</h3>
+<p>
+  Vespa containers are in general using "container" as their service name. However, a container cluster that has
+  declared neither <a href="reference/services-docproc.html">document-processing</a> nor
+  <a href="reference/services-container.html#document-api">document-api</a> has been named "qrserver".
```

Comment:
suggested language change "has been named" -> "used to be named"

Reply:
Thanks, done.